packages/components/src/components/hds/app-footer/index.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+import type { ComponentLike } from '@glint/template';
+import type { HdsYieldSignature } from '../yield';
+import type HdsAppFooterItemSignature from './legal-links';
+import type HdsAppFooterLegalLinksSignature from './legal-links';
+import type HdsAppFooterLinkSignature from './link';
+import type HdsAppFooterStatusLinkSignature from './status-link';
+
+export interface HdsAppFooterSignature {
+  Args: {
+    ariaLabel?: string;
+    copyrightYear?: string;
+    theme?: string;
```

Comment:
this could as well be an enum of `light`, `dark` (placed in `types.ts`)

Reply:
Good idea. Added in b088c4cdbeb268ebc88ae8855a1bb3995eeb3ba8